scenario_player/tasks/base.py

```diff
@@ -82,6 +83,7 @@ class Task:
     #
     # Ref.: https://github.com/raiden-network/raiden/issues/6149#issuecomment-627387624
     SYNCHRONIZATION_TIME_SECONDS = 30
```

Comment:
Do we still need this?

Reply:
I set it to 0 for now, once the scenarios run fine I'm going to remove it.